src/test/java/com/google/api/codegen/testdata/python_doc_versioned_directory__init___no_path_templates.baseline

```diff
@@ -0,0 +1,31 @@
+============== file: google/cloud/proto/example_v1/__init__.py ==============
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import absoulte_import
+
+from example.helpers import LibraryHelpers
+from google.cloud.proto.example.v1 import no_path_templates_pb2
+from google.cloud.proto.example_v1 import types
+
+
+class NoTemplatesAPIServiceClient(LibraryHelpers, no_path_templates_pb2.NoTemplatesAPIServiceClient):
```

Comment:
`LibraryHelpers` is manually written and likely should not be referenced by autogen.